addon/components/frost-toggle.js

```diff
@@ -130,8 +163,8 @@ export default Component.extend(PropTypeMixin, FrostEventsProxy, {
       }
     },
     /* eslint-enable complexity */
-    noop () {
-      // we have this method to avoid action undefined error
-    }
+
+    /** A no-op action to avoid an action undefined error */
```

Comment:
OK, I was  trying to be nice and add the missing JSDoc comment, I guess I should have just gone with
```js
// FIXME: jsdoc
```

On previous teams, I've had people complain about the verbosity of ```js
/**
 *
 */
```
for single-line function descriptions. I guess I just can't win.